### ISSUE TITLE
Latest commits not compiling.

### ISSUE THREAD

**Issue by andrewkm** [Nathat23/StackMob-5#204]
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/StackMob-5/src/main/java/uk/antiperson/stackmob/StackMob.java:[98,43] cannot find symbol
  symbol:   class SimplePie
  location: class org.bstats.bukkit.Metrics
[ERROR] /var/lib/jenkins/workspace/StackMob-5/src/main/java/uk/antiperson/stackmob/StackMob.java:[99,20] cannot find symbol
  symbol:   method isEnabled()
  location: variable metrics of type org.bstats.bukkit.Metrics
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  19.252 s
[INFO] Finished at: 2021-04-03T07:40:34-04:00
[INFO] ------------------------------------------------------------------------
Waiting for Jenkins to finish collecting data
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project StackMob: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/StackMob-5/src/main/java/uk/antiperson/stackmob/StackMob.java:[98,43] cannot find symbol
[ERROR] symbol:   class SimplePie
[ERROR] location: class org.bstats.bukkit.Metrics
[ERROR] /var/lib/jenkins/workspace/StackMob-5/src/main/java/uk/antiperson/stackmob/StackMob.java:[99,20] cannot find symbol
[ERROR] symbol:   method isEnabled()
[ERROR] location: variable metrics of type org.bstats.bukkit.Metrics
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by Geolykt** [comment]
I don't think that this is the full error, is there anything beforehand?

**Comment by Nathat23** [comment]
Fixed in d4275e16a05abcbeeea404369f46f2f4604dff7f

**Comment by andrewkm** [comment]
Thanks! :)
